ckan/public/base/css/main-rtl.css

```diff
@@ -13642,13 +13642,13 @@ td.diff_header {
 .js .table-toggle-more .show-less {
   display: none;
 }
-.js .table-toggle-more .toggle-seperator {
+.js .table-toggle-more .toggle-separator {
   display: table-row;
 }
-.js .table-toggle-more .toggle-seperator td {
+.js .table-toggle-more .toggle-separator td {
   height: 11px;
   padding: 0;
-  background-image: url("../../../base/images/table-seperator.png");
+  background-image: url("../../../base/images/table-separator.png");
```

Comment:
just now mentioned this. Pushed separate ea3c259274c5b5d2db7afd9df06d82e39c420cfe to rename the file as well.